torchrl/envs/libs/vmas.py

```diff
@@ -115,13 +115,20 @@ class VmasWrapper(_EnvWrapper):
         env (``vmas.simulator.environment.environment.Environment``): the vmas environment to wrap.
 
     Keyword Args:
-        num_envs (int): Number of vectorized simulation environments.
-        device (torch.device, optional): Device for simulation.
-        continuous_actions (bool, optional): Weather to use continuous actions. Defaults to ``True``.
-        max_steps (int, optional): Maximum number of steps in each vectorized environment after which done is returned.
-            Defaults to ``None`` (no truncation).
+        num_envs (int): Number of vectorized simulation environments. VMAS perfroms vectorized simulation using PyTorch.
+            This argument decides the number of vectorized envuironments that should be simulated in a batch. It will also
+            determine the batch size of the environment.
+        device (torch.device, optional): Device for simulation. Defaults to ``"cpu"``. All the tensors created by VMAS
+            will be placed on this device.
+        continuous_actions (bool, optional): Whether to use continuous actions. Defaults to ``True``. If ``False``, actions
+            will be discrete. The number of actions and their size will depend on the scenario chosen.
+            See the VMAS repositiory for more info.
+        max_steps (int, optional): Horizon of the task. Defaults to ``None`` (infinite horizon). Each VMAS scenario can
+            implement a ``done`` function that will define when the scenario is terminated. If ``max_steps`` is specified,
+            the scenario will also be terminated after this horizon has been reached. If instead of terminating the scenario
+            you wish to truncate it, please use a :class:`~torchrl.envs.transforms.StepCounter` transform.
```

Comment:
I don't get the distinction except that StepCounter sets a truncation. It's not clear to which extend they differ from each other.

Reply:
max_steps stets the internal vmas done (terminated in torchrl). StepCounter would set truncation